godotopenxrmeta/src/main/cpp/export/meta_export_plugin.cpp

```diff
@@ -104,6 +104,46 @@ MetaEditorExportPlugin::MetaEditorExportPlugin() {
 			false,
 			false
 	);
+	_use_scene_api_option = _generate_export_option(
+			"meta_xr_features/use_scene_api",
+			"",
+			Variant::Type::BOOL,
+			PROPERTY_HINT_NONE,
+			"",
+			PROPERTY_USAGE_DEFAULT,
+			false,
+			false
+	);
```

Comment:
This can be removed since it's being added in https://github.com/GodotVR/godot_openxr_vendors/pull/66